### ISSUE TITLE
Polling Interval, Less than 5 Seconds Possible?

### ISSUE THREAD

**Issue by MediumFidelity** [dave92082/SolarEdge-Exporter#4]
More of a question than an issue, but what defines the 5 second polling interval?  The OLED display on the inverter appears to update at ~2Hz.  Curious if you tried after polling and ran into issues.

https://github.com/dave92082/SolarEdge-Exporter/blob/5e176ce9fdfe873baa61e6a01e82af3e05f3a97b/main.go#L118

**Comment by dave92082** [comment]
The interval is located in the configuration file `config.yaml`. The Inverter will respond much faster and in my testing it was responding in about 10 milliseconds. The main reason I limited the duration to a multiple of seconds rather than milliseconds was that too many requests locked up the network stack on my inverter and it would not come back until I disconnected the Ethernet and re-connected.
